3-enrich/stream-enrich/project/Dependencies.scala

```diff
@@ -36,20 +36,21 @@ object Dependencies {
     val awsSdk               = "1.6.11"
     val kinesisClient        = "1.6.1"
     val kafkaClients         = "0.10.1.0"
+    val nsqClient            = "1.0.0.RC4"
     // Scala
-    val argot                = "1.0.1"
-    val config               = "1.0.2"
+    val argot                = "1.0.3"
+    val config               = "1.3.1"
     val scalaUtil            = "0.1.0"
     val snowplowRawEvent     = "0.1.0"
-    val snowplowCommonEnrich = "0.24.0"
+    val snowplowCommonEnrich = "0.25.0"
     val scalazon             = "0.11"
-    val scalaz7              = "7.0.0"
-    val igluClient           = "0.4.0"
+    val scalaz7              = "7.0.9"
+    val igluClient           = "0.5.0"
     val snowplowTracker      = "0.3.0"
     // Scala (test only)
     // val specs2               = "2.3.6" Conflicts with com.chuusai:shapeless
-    val specs2               = "2.2.3"
-    val scalazSpecs2         = "0.1.2"
+    val specs2               = "2.3.11"
+    val scalazSpecs2         = "0.2"
```

Comment:
please don't change library versions if you don't need to. this will be taken care of separately